hack/test-cmd.sh

```diff
@@ -38,22 +41,23 @@ function start-proxy()
 {
   stop-proxy
 
-  kube::log::status "Starting kubectl proxy"
+  kube::log::status "Starting kubectl proxy on random port"
 
-  for retry in $(seq 1 3); do
-    PROXY_PORT=$(kube::util::get_random_port)
-    kube::log::status "On try ${retry}, use proxy port ${PROXY_PORT} if it's free"
-    if kube::util::test_host_port_free "127.0.0.1" "${PROXY_PORT}"; then
-      if [ $# -eq 0 ]; then
-        kubectl proxy -p ${PROXY_PORT} --www=. 1>&2 & break
-      else
-        kubectl proxy -p ${PROXY_PORT} --www=. --api-prefix="$1" 1>&2 & break
-      fi
-    fi
-    sleep 1;
+  if [ $# -eq 0 ]; then
+    kubectl proxy --port=0 --www=. 1>proxy-port.out &
```

Comment:
wonder if we should use `mktemp` so there's no chance of colliding with another parallel run?